src/hooks/useOverlay.ts

```diff
@@ -0,0 +1,17 @@
+import { ConfirmProps } from 'components/OverlaysProvider/OverlayConfirm';
+import { OverlayContext } from '../components/OverlaysProvider';
+import { useContext } from 'react';
+
+interface UseOverlayReturnProps {
+    showConfirm: (props: Omit<ConfirmProps, 'open'>) => void;
+}
+
+export default function useOverlay(): UseOverlayReturnProps {
+    const overlay = useContext(OverlayContext);
+
+    return {
+        showConfirm: (props: Omit<ConfirmProps, 'open'>) => {
+            overlay.setConfirmProps({ open: true, ...props });
+        },
+    };
+}
```

Comment:
I don't think we need this as a discrete hook, the context can expose these functions in the same way I'm doing in my pr #529 .